python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -11,7 +11,7 @@
 
 class DagsterDbtTranslator:
     @classmethod
-    def get_asset_key(cls, dbt_resource_info: Mapping[str, Any]) -> AssetKey:
+    def get_asset_key(cls, dbt_repsource_props: Mapping[str, Any]) -> AssetKey:
```

Comment:
Assume this is a typo - could we also update the docstrings to use `properties` instead of `information` and also give a durable link to the dbt manifest API (https://docs.getdbt.com/reference/artifacts/manifest-json) when documenting the argument?